e2e-cypress/files/package.json

```diff
@@ -21,12 +21,15 @@
   "devDependencies": {
     "@types/node": "^22.4.1",
     "cypress": "^13.13.1",
+    "cypress-multi-reporters": "^1.6.4",
     "junit-report-merger": "^7.0.0",
     "mocha-junit-reporter": "^2.2.1",
     "npm-run-all": "^4.1.5",
     "otplib": "^12.0.1",
+    "puppeteer": "^23.5.1",
     "rimraf": "^6.0.1",
     "sharp": "^0.33.5",
+    "ts-node": "^10.9.2",
```

Comment:
I saw that `ts-node` is used with `npx` inside the `Jenkinsfile.template`. Either it should be removed here or maybe there should be a dedicated PDF task as part of `package.json` that is then called by `npm` instead of `npx`.